qiime2/core/format.py

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import qiime2.core.path as qpath
+
+
+class FormatBase:
```

Comment:
this just got moved up to keep import dependency clean

Reply:
👍